addons/storyquest_bootstrap/plugin.gd

```diff
@@ -10,24 +10,37 @@ const NEW_STORYQUEST_DIALOG = preload(
 const TOOL_MENU_LABEL := "Create StoryQuest from template..."
 
 const STORYQUESTS_PATH := "res://scenes/quests/story_quests/"
+const TEMPLATE_PREFIX := "NO_EDIT"
 # Using + instead of String.path_join() here because it errors with:
 # Assigned value for constant "TEMPLATE_PATH" isn't a constant expression.
-const TEMPLATE_PATH := STORYQUESTS_PATH + "template/"
+const TEMPLATE_PATH := STORYQUESTS_PATH + TEMPLATE_PREFIX + "/"
+const QUEST_FILENAME := "quest.tres"
 const MIN_TITLE_LENGTH := 4
 
 var template_info := {
 	"intro":
 	{
-		"scene_path": "/0_template_intro/template_intro.tscn",
-		"components_path": "/0_template_intro/template_intro_components/",
-		"dialogue_path": "/0_template_intro/template_intro_components/template_intro.dialogue",
-		"image_path": "0_template_intro/template_intro_components/template_intro_image.png",
+		"scene_path": "/0_{prefix}_intro/{prefix}_intro.tscn".format({"prefix": TEMPLATE_PREFIX}),
```

Comment:
I personally think that the repeated `.format({"prefix": TEMPLATE_PREFIX})` is much less legible than repeating `NO_EDIT`, even at the cost having to search-and-replace every line if we rename it again. But let's go with this for now.